dev-libs/qtcompress/qtcompress-0_p20180513.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils toolchain-funcs virtualx
+
+DESCRIPTION="A public API for QZip in an easy to use module"
+
+HOMEPAGE="https://github.com/nezticle/qtcompress"
+MY_COMMIT="23f8831826cd72aedf99fc3699148b6c994fd677"
+SRC_URI="https://github.com/nezticle/qtcompress/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+LICENSE="LGPL-2.1"
```

Comment:
The upstream Git repo has more licenses listed. I know it bundles some stuff and some of the licenses could be related to that, but looking at 
https://github.com/nezticle/qtcompress/blob/master/src/compress/qzip.cpp
it lists a choice of different licenses being used here.

Now I am **not** a license expert but looks to me this could be a case for `LICENSES="|| ( 1 2 3 )` and maybe a comment above it.